scripts/dts/test-bindings/device-on-any-bus.yaml

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Is the license here correct?

Reply:
It’s a copy-paste of the other yaml files in this directory, but I’m also surprised to see that it’s not Apache 2. @mbolivar-nordic @galak ?

Reply:
@henrikbrixandersen upon closer inspection the entire Zephyr DT implementation (i.e. everything under `scripts/dts`) seems to be BSD 3-clause, so if you think there's a concern with that it needs to be addresses independently to this PR. With that in mind, do you want to reconsider your "changes requested"? :)